README.md

```diff
@@ -107,12 +104,28 @@ Default options:
 
 ```lua
 {
-  auto_enable = true, -- Enable RustOwl immediately
+  auto_attach = true, -- Auto attach the RustOwl LSP client when opening a Rust file
+  auto_enable = false, -- Enable RustOwl immediately when attaching the LSP client
   idle_time = 500, -- Time in milliseconds to hover with the cursor before triggering RustOwl
-  client = {}, -- LSP client configuration that gets passed to `require('lspconfig').rustowlsp.setup()`
+  client = {}, -- LSP client configuration that gets passed to `vim.lsp.start`
 }
 ```
 
+When opening a Rust file, the Neovim plugin creates the `Rustowl` user command:
+
+```vim
+:RustOwl {subcommand}
```

Comment:
Not sure if `:RustOwl` or `:Rustowl` would be better.
The former might be easier to find in tab completion suggestions, but the latter is one less key press.

Reply:
I think `:RustOwl` is good here for consistency